doc/sphinx/administration_portal/client/vpbx/routing_endpoints/conference_rooms.rst

```diff
@@ -25,6 +25,9 @@ The following image shows the process of creating a new conference room:
         Maximum number of participants in the conference. When this limit is 
         reached, join requests will be rejected.
 
+    Announce user count
+        First member or allways
```

Comment:
Typo _always_

You can also add: _Play a locution when joining a conference telling how many users there are already participating. Options are: only to the first member of the conference or anybody joining._